content/python/concepts/strings/strings.md

```diff
@@ -71,17 +71,17 @@ But here, upon this bank and shoal of time,
 We'ld jump the life to come."""
 ```
 
-## Escape Character
+## Escape Characters
 
-Sometimes a string may have a character that Python tries to interpret such as `'`.
+Sometimes a string may have a character that Python tries to interpret, such as `'`.
 
 ```py
 my_string = 'It's a lovely day!'
 
 print(my_string)
 ```
 
-This will give an error because the interpreter thinks that the second `'` marks the end of the string.
+This will raise an error, because the interpreter thinks the second `'` marks the end of the string.
```

Comment:
```suggestion
This will raise an error because the interpreter marks the successive single quote `'` as the end of the string.
```

Like the give-> raise substitution. Minor changes to the second half 📝

Reply:
Thanks. I understand "successive" is the technical term, but I think using it might put off some novice readers. Perhaps instead we could say something like this:

>marks the single quote intended to appear as an apostrophe within the string as the end of the string instead.

That may be a bit long-winded. Open to discussing it further.

Reply:
In fact, could we try:

> This will raise an error because Python interprets the single quote (`'`) intended to be an apostrophe as the end of the string instead.

What do you think?

Reply:
Yup works for me! Slapped some commas in there -> This will raise an error because Python interprets the single quote ('), intended to be an apostrophe, as the end of the string.